x/dex/types/trading_pair.go

```diff
@@ -134,6 +135,7 @@ func (p *TradingPair) UpdateTickPointersPostRemoveToken1(goCtx context.Context,
 
 func (p TradingPair) FindNextTick1To0(goCtx context.Context, k Keeper) (tickIdx int64, found bool) {
```

Comment:
Seems like it would be better to restructure the usage of this function to use the same iterator that exhausted the previous tick of liquidity to determine the "NextTick1To0"